src/bundle/Resources/public/scss/_filters.scss

```diff
@@ -46,6 +48,29 @@
         }
     }
 
+    &:before,
+    &:after {
+        content: '';
+        position: absolute;
+        border-style: solid;
+        border-width: 0 calculateRem(10px) calculateRem(10px);
+        display: block;
+        width: 0;
+        z-index: 1;
+    }
+
+    &:before {
+        border-color: $ez-color-secondary transparent;
+        top: calculateRem(-10px);
+        left: 93.75%;
```

Comment:
This property can be moved to the aggregated selector: `&:before, &:after {`

Reply:
Ok.